scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -888,6 +888,7 @@
         # without first triggering the moveto event guarding him (see farther below). Add these events as backups:
         [event]
             name=attack
+            first_time_only=no
```

Comment:
I have a feeling that having this happen every time you attack him would look kind of jank.

Reply:
Right I'm saying the spawning additional guards seems like it would be kind of jank. Imagine surrounding him with say 3-4 units, you attack him and in a single turn a bunch of guards spawn. That would just be a bit awkward. If the player is good enough to kill him then just let the player kill him imo.

Reply:
> Right I'm saying the spawning additional guards seems like it would be kind of jank. Imagine surrounding him with say 3-4 units, you attack him and in a single turn a bunch of guards spawn. That would just be a bit awkward. If the player is good enough to kill him then just let the player kill him imo.

Hm, how about a compromise where I just reduce the number of guards spawned?

Reply:
It's not the number of guards, it's also the behavior of spawning a guard each time you attack him, I think that'd just look hilarious and probably a bit awkward. Instead you could maybe spawn more guards on the first time.

Reply:
> It's not the number of guards, it's also the behavior of spawning a guard each time you attack him, I think that'd just look hilarious and probably a bit awkward. Instead you could maybe spawn more guards on the first time.

Hm, maybe just update the conditional so the case for the second time doesn't happen as much? (I reduced the number anyways in [0a67b21](https://github.com/nemaara/A_New_Order/pull/76/commits/0a67b2107912a35e461c99984e7cbd61e3068a69) btw)

Reply:
> Hm, maybe just update the conditional so the case for the second time doesn't happen as much?

OK I did that in [3a514c3](https://github.com/nemaara/A_New_Order/pull/76/commits/3a514c31dd9db41ed64e37fc7a31ca943d78321f)
(side note, but I kinda wish the `[variable]` tag supported an `is_in_range=` key... maybe there's a way to do that in WFL; I'll have to ask celmin over the Discord/IRC bridge...)

Reply:
Hmm another idea would be to make it trigger at most once per turn and have some text each time? This way you won't get the odd case where it randomly spawns guards on you without warning. I'm not sure I understand the behavior as you have it in the latest revision.

Reply:
> Hmm another idea would be to make it trigger at most once per turn and have some text each time? This way you won't get the odd case where it randomly spawns guards on you without warning. I'm not sure I understand the behavior as you have it in the latest revision.

Current code as of latest revision just has it trigger once more after the first time, so you get: first time = Darknites + Healer, second time = Protectors, third time (and any time later) = nothing.